Rendera/include/camera.hpp

```diff
@@ -16,7 +16,7 @@ class Camera{
     //viewport screen at (x = focal_length) (viewing direction is +X)
     
     public:
-
+        Camera(){}
```

Comment:
Is this default constructor really needed? Also, if we must have it, why not do `Camera() = default;` instead of having am empty body instead?

Same comment for the default constructor you added in the `Material` class.